profiles/suparn_nayak.md

```diff
@@ -0,0 +1,14 @@
+# first and last name
```

Comment:
Just replace this with `# Supran Nayak`
- Delete the 2nd line with your name
-  Just push another commit with the changes to override this